src/platform/packages/shared/kbn-grouping/src/hooks/use_grouping.tsx

```diff
@@ -10,11 +10,11 @@
 import { FieldSpec } from '@kbn/data-views-plugin/common';
 import React, { useCallback, useMemo, useReducer } from 'react';
 import { UiCounterMetricType } from '@kbn/analytics';
-import { groupsReducerWithStorage, initialState } from './state/reducer';
+import { groupsReducerWithStorage, initialState as reducerinitialGroupings } from './state/reducer';
 import { GroupingProps, GroupSelectorProps, isNoneGroup } from '..';
```

Comment:
@baileycash-elastic nit: It would be nice to use `camelCase` syntax here `reducerInitialGroupings`